client/signup/steps/site-title/style.scss

```diff
@@ -0,0 +1,81 @@
+.signup-site-title {
```

Comment:
The `SignupSiteTitle` component has been extracted out of the step itself and the style moved, so the rule is more relevant now.